src/assets/internal/icons/index.web.ts

```diff
@@ -0,0 +1,47 @@
+export const icons = {
```

Comment:
Is the separation for index.web file is because the web file contain dimensions?